worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/processor/MultiBatchProcessor.java

```diff
@@ -80,28 +77,18 @@ public void removeBatchProcessor(IBatchProcessor processor) {
 
     @Override
     public IChunkSet processSet(IChunk chunk, IChunkGet get, IChunkSet set) {
-        Map<Integer, Set<IBatchProcessor>> ordered = new HashMap<>();
+        Map<Integer, List<IBatchProcessor>> ordered = new HashMap<>();
```

Comment:
Could it be worth caching this per processor and marking dirty when changed?

Reply:
I tried to profile that code, but I got like 3 traces in that code vs thousands in the `processSet` calls below. So it's not really even measurable yet noticeable.

The change from HashSet to ArrayList should simplify insertion and iteration logic, but given the measurement difficulty, that's rather theoretical.

Reply:
Hmm fair enough. Should have been cached in the first place really but I suppose it doesn't have any noticeable performance impact